### ISSUE TITLE
mnemonic passphrase feature requires missing method

### ISSUE THREAD

**Issue by gz-c** [skycoin/hardware-wallet#115]
https://github.com/trezor/trezor-crypto/blob/b7e99aa76c36417b797add2addba03121b758e66/bip39.c#L188

The method linked above is required to use the passphrase feature of mnemonic generation.

This method was removed from our source tree. Restore the method.  The bip39 library should probably be restored in full, from the trezor sources. Unless there is a good reason to modify it.

**Comment by olemis** [comment]
This is already in source tree 

```sh
$ grep -nr mnemonic_to_seed .
./tiny-firmware/vendor/skycoin-crypto/trezor-tools/bip39.h:40:void mnemonic_to_seed(const char *mnemonic, const char *passphrase, uint8_t seed[512 / 8], void (*progress_callback)(uint32_t current, uint32_t total));
./skycoin-api/trezor-tools/bip39.h:40:void mnemonic_to_seed(const char *mnemonic, const char *passphrase, uint8_t seed[512 / 8], void (*progress_callback)(uint32_t current, uint32_t total));
```

**Comment by olemis** [comment]
`./tiny-firmware/vendor/skycoin-crypto/trezor-tools` is a symlink to `./skycoin-api/trezor-tools` .

**Comment by olemis** [comment]
`bip39.c` file is missing

**Comment by gz-c** [comment]
This is not assigned to the release milestone so does not need action at this time

**Comment by olemis** [comment]
fixed by d1b8fa3
